tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AppSec.cs

```diff
@@ -81,6 +81,12 @@ internal class AppSec
             /// </summary>
             internal const string ApiSecurityRequestSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE";
 
+            /// <summary>
+            /// Configuration key for the maximum number of requests
+            /// to be analyzed by api security concurrently. Defaults to 1.
+            /// </summary>
+            internal const string ApiSecurityMaxConcurrentRequests = "DD_API_SECURITY_MAX_CONCURRENT_REQUESTS";
```

Comment:
You'll need to add this to the config_norm_rules and make a PR to `dd-go` 🙂

Reply:
thanks added it here https://github.com/DataDog/dd-go/pull/117134
and in bca72f692faf563e74bd285e3d25c427ffc78f5d